src/components/autocomplete/Autocomplete.vue

```diff
@@ -123,6 +123,15 @@ export default {
                     whiteList.push(child)
                 }
             }
+            if (this.$parent.$options.name === 'BTaginput') {
```

Comment:
Please use `_isTaginput` to check the component

Reply:
_isTaginput is not available via `this.$parent._isTaginput`

Reply:
`this.$parent.$data._isTaginput` is ok?

Reply:
Sure